### DISCUSSION TITLE
Not popping up

### DISCUSSION THREAD

**Discussion by Tetsyuba1** [3kh0/ext-remover#474] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have tried killcurly that didnt work, LTBEEF including DNS but whenever I click the bookmark nothing ever pops up. 

i also cant use inspect
my version is Version 105.0.5195.134 (Official Build) (64-bit)
I have goguardian and im on home wifi

**Comment by Aka-but-nice** [comment, marked as answer]
You probably have bookmarklets disabled, which may be the reason why ltbeef isn't working, consider checking your policies to see if JavaScript is blocked, they may help you understand and securly is really the only one that killcurly works best, others are a little worst.

**Reply by Tetsyuba1** [reply]
dammnn so that means theres nothing i can do?   🥲

**Reply by Aka-but-nice** [reply]
Well have you checked, look at "URLBlockList" in policies, click show more and if you see something like javascript:// then it is blocked.
